### PR TITLE
remove angular foundation and use only ionic modals

### PR DESCRIPTION
**Pull request by gabrielbazan7**
fixes : #4395

### CODE REVIEW

File: src/js/controllers/modals/txStatus.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+'use strict';
+
+angular.module('copayApp.controllers').controller('txStatusController', function($scope, $timeout, profileService) {
```
Comment (JDonadio) [inline src/js/controllers/modals/txStatus.js]: profileService is not used

**Comment by cmgustavo** [comment]
Scann QR only works the first time. If you open it again, it doesn't work.

**Comment by cmgustavo** [comment]
Except the QR scan, LGTM.

**Comment by gabrielbazan7** [comment]
Updated

**Comment by gabrielbazan7** [comment]
Updated

**Comment by cmgustavo** [comment]
ACK

**Pull request merged**